makefile

```diff
@@ -12,8 +12,8 @@ LIBS     = -static-libgcc -m64 -g3 -ltermcap -lncurses -lpthread -lm
 CLIBS    = -static-libgcc -m64 -g3
 INCS     =
 
-QRACKLIBS= qrack/build/libqrack.a
-QRACKINCS= -Iqrack/include
+QRACKLIBS= -Lqrack/build -lqrack
+QRACKINCS= -Iqrack/include -Iqrack/include/common -Iqrack/build/include 
```

Comment:
Unnecessary; files in common should be included in qrack as "common/...h"

Reply:
Also, switching to -l doesn't always behave as you would like and isn't appropriate here.

Reply:
Done.